chat-server/app/utils/graph_utils/table_utils.py

```
@@ -0,0 +1,58 @@
+    Args:
+        table_name: Name of the table
+        org_id: Optional organization ID for multi-tenant support
+
+    Returns:
+        Estimated row count, or 0 if table not found or error occurs
+    """
+    size_query = f"""
+        SELECT estimated_size
+        FROM duckdb_tables()
+        WHERE table_name = '{table_name}'
+    """
```

Ruff (0.14.13)

Possible SQL injection vector through string-based query construction

(S608)